docs/connect/index.md

```diff
@@ -118,11 +167,17 @@ An HTTP URL to visit Admin UI.
 
 
 ```{tip}
-- Specify the [schema] name `doc`, if you are asked for a *database name*.
-- The default [superuser] on a vanilla CrateDB cluster is called `crate`,
-  without a password.
-- When aiming to authenticate properly, please learn about the available
-  [authentication] options. 
+- CrateDB's fixed catalog name is `crate`, the default schema name is `doc`.
+- CrateDB does not implement the notion of a database,
+  however tables can be created in different [schemas].
+- When asked for a *database name*, specifying a schema name (any),
+  or the fixed catalog name `crate` may be applicable.
+- If a database-/schema-name is omitted while connecting,
+  the PostgreSQL drivers may default to the "username".
+- The predefined [superuser] on an unconfigured CrateDB cluster is
+  called `crate`, defined without a password.
+- For authenticating properly, please learn about the available
+  [authentication] options.
 ```
```

Comment:
### Reply
>> Tools like DBeaver will default to your username as "database" name. In CrateDB this currently sets your searchpath to username. Do you want that?
>
> This is certainly not intended, [...] improving the documentation [makes sense]. 

Thanks again. Within the "Tip" section below, I've added this item:
> - If a database-/schema-name is omitted while connecting,
>   the PostgreSQL drivers may default to the "username".

### Report
There have been a few more updates to this section, aiming to compress relevant details about those matters into a single spot.

![image](https://github.com/user-attachments/assets/a85b8af3-6a6c-4f63-8612-e3087f51da2a)
-- [Connect » Configure](https://crate-clients-tools--151.org.readthedocs.build/en/151/connect/index.html#configure)

**Please let us know if you can think of any other items missing, or if existing ones need to improve.**

Reply:
We compiled those details based on a few empirical exercises, and your advises.

**Please let us know if you can think of any other items missing, or if existing ones need to improve.**

Reply:
I guess it will not harm to merge this. Please let me know about any improvements and wished also in retrospective. Thanks for your valuable support so far!